packages/algolia-search/plugins/AlgoliaSearchContextPlugin.tsx

```diff
@@ -23,7 +23,7 @@ function AlgoliaSearchContextPlugin(props: PluginProps<SearchContextProps>) {
   return (
     <InstantSearchSSRProvider {...(typeof serverProps === 'object' ? serverProps : {})}>
       <InstantSearch searchClient={searchClient} indexName={searchIndex}>
-        <Prev {...rest} />
+        {isNext && <Prev {...rest} />}
```

Comment:
This is the only place I'm seeing isNext, not sure what this is?

Reply:
I've renamed this prop to `rendersInsideNextjs`. We hydrate the `InstantSearchSSRProvider` in the getStaticProps by rendering the component with the `getServerState` function of Algolia. Since almost all components use `useRouter`, which requires the components to be rendered by NextJS, we don't render those components if rendered by the `getServerState` function.

The short answer is, this allows the server to hydrate the Algolia context